crates/core/tedge_actors/src/errors.rs

```diff
@@ -1,6 +1,7 @@
 use thiserror::Error;
 use tokio::task::JoinError;
 
+/// Box dyn errors: wrapping errors sent by actors to the runtime
```

Comment:
```suggestion
/// A wrapper for runtime errors raised from actors to the runtime.
/// Boxing is required as the errors of different actors would be of different sizes.
```

Reply:
Updated - even if not convinced that this is the place to explain the purpose of boxing.